### PR TITLE
add NUMERIC dtype to parse_bg_type

### PR DESCRIPTION
**Pull request by paulsendavidjay**
returns BQ_FLOAT

### CODE REVIEW

**Comment by paulsendavidjay** [comment]
addresses issue
https://github.com/r-dbi/bigrquery/issues/282

**Comment by hadley** [comment]
Can you please add a bullet to NEWS? It should briefly [describe the change](http://style.tidyverse.org/news.html) and end with `(@yourname, #issuenumber)`.

**Comment by paulsendavidjay** [comment]
Not sure if I addressed correctly. I hope so.

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # bigrquery 1.0.0.9000
 
+* In parse_bq_type(), x now evaluates 'NUMERIC' as 'BQ_FLOAT' rather than throwing error. (@paulsendavidjay, #282, #286)
+
+* Unparseable date times return NA (#285)
```
Comment (hadley) [inline NEWS.md]: You seem to have ended up with extra news entries. Can you please delete?

**Comment by hadley** [comment]
Thanks!

**Pull request merged**